test/runtests.jl

```diff
@@ -1,5 +1,11 @@
 using CCDReduction
 using Test
 using Statistics
+using FITSIO
+
+const test_file_url_M6707HH = "https://www.cfa.harvard.edu/~dfabricant/huchra/ay16/fits/M6707HH.fits"
+const test_file_path_M6707HH = joinpath(@__DIR__, "MH6707HH.fits")
+!isfile(test_file_path_M6707HH) && download(test_file_url_M6707HH, test_file_path_M6707HH)
```

Comment:
For the CI to cache we should give it a _directory_. we don't want to have it cache our entire test directory, though, so I suggest this
```suggestion
const test_file_path_M6707HH = joinpath(@__DIR__, "data", "MH6707HH.fits")
if !isfile(test_file_path_M6707HH)
	mkpath(dirname(test_file_path_M6707HH))
	download(test_file_url_M6707HH, test_file_path_M6707HH)
end
```

Reply:
Also, you need to add the cache directive to travis:
```
cache:
    directories:
		- tests/data
```